frontend/src/components/welcome/landing-page.js

```diff
@@ -0,0 +1,224 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import peopleImage from '../../images/landingPage_People.png'
+import logoGrey from '../../images/logo-complete-gray.png'
+import Grid from '@material-ui/core/Grid'
+// import LoginButton from "../session/login-button";
+// import DialogTitle from '@material-ui/core/DialogTitle';
+// import Dialog from '@material-ui/core/Dialog';
```

Comment:
What about these commented lines @Shawn-Noruzi?

Reply:
I left those in case I was going to be implementing the login functionality. 

Thinking back though I think removing it would have made the PR cleaner

--- I can remove and re-PR if needed

Reply:
@alexandreramosdev 
Sounds great - removing the comments and will resolve once done!

Reply:
Great @Shawn-Noruzi, just push to the same branch and the PR will be updated 👍